darts/utils/torch.py

```diff
@@ -20,6 +23,45 @@
 MAX_NUMPY_SEED_VALUE = (1 << 31) - 1
 
 
+class MonteCarloDropout(nn.Dropout):
+    """
+    Defines Monte Carlo dropout Module as defined
+    in the paper https://arxiv.org/pdf/1506.02142.pdf.
+    In summary, This technique uses the regular dropout
+    which can be interpreted as a Bayesian approximation of
+    a well-known probabilistic model: the Gaussian process.
+    We can treat the many different networks
+    (with different neurons dropped out) as Monte Carlo samples
+    from the space of all available models. This provides mathematical
+    grounds to reason about the model’s uncertainty and, as it turns out,
+    often improves its performance.
+    """
+
+    # We need to init it to False as some models may start by
+    # a validation round, in which case MC dropout is disabled.
+    mc_dropout_enabled: bool = False
+
+    def train(self, mode: bool = True):
```

Comment:
I suggest renaming this to `train`. `mode` is not descriptive as a bool

Reply:
I agree, unfortunately this is coming from PyTorch and we have to follow the nomenclature here, because we are inheriting `nn.Dropout` and this function might be called from PyTorch using `mode` as an argument name.